homeassistant/components/blueprint/websocket_api.py

```diff
@@ -84,3 +87,82 @@ async def ws_import_blueprint(hass, connection, msg):
             },
         },
     )
+
+
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
```

Comment:
You should pass in the url that it was fetched from, and we should store that in the metadata. That way we can check for updates in the future.